Assets/GothicVR/Scripts/Player/Climb/RayTeleportClimb.cs

```diff
@@ -150,6 +150,7 @@ private void CreateReticle()
             reticleImage.type = Image.Type.Filled;
             reticleImage.fillMethod = Image.FillMethod.Vertical;
             Destroy(reticle.GetComponent<Collider>());
+            reticle.SetActive(false);
```

Comment:
I know it's not the topic for your branch, but can we please rename this one to something meaningful? ;-)